packer-scripts/pre-chef-bootstrap

```diff
@@ -85,6 +85,8 @@ __setup_sshd() {
   if ! grep -Eq '^sshd: ALL: ALLOW' /etc/hosts.allow; then
     echo 'sshd: ALL: ALLOW' >>/etc/hosts.allow
   fi
+
+  sed -i 's/PasswordAuthentication without-password/PasswordAuthentication no/g' /etc/ssh/sshd_config
```

Comment:
This is not where this change should be made, but instead in `./cookbooks/travis_internal_base/attributes/default.rb`